events/ME_Canada_Events.txt

```diff
@@ -931,7 +931,6 @@ country_event = {
 	picture = REVOLT_eventPicture
 
 	fire_only_once = yes
-	is_triggered_only = yes
```

Comment:
Searched for `ME_Canada_Events.17` and did not find an event call for it in the mod files.

![image](https://user-images.githubusercontent.com/5704491/91648590-08b46e80-ea05-11ea-9311-e42fc22eeb49.png)